auth-method-oauth2.js

```diff
@@ -1905,6 +1905,7 @@ class AuthMethodOauth2 extends AmfHelperMixin(AuthMethodBase) {
           <anypoint-button
             ?disabled="${_authorizing}"
             class="auth-button"
+            compatibility="${compatibility}"
```

Comment:
In both cases, boolean attributes must be set with `?` prefix per `LitElement` documentation (`lit-html` to be exact). With this, it always renders this button in compatibility mode as this attribute would be always set.

Reply:
Please, see https://lit-element.polymer-project.org/guide/templates#bind-to-a-boolean-attribute